spec/system/casework/reviewing/a_pse_application_spec.rb

```diff
@@ -4,21 +4,71 @@
   include_context 'with stubbed application'
 
   let(:application_id) { '21c37e3e-520f-46f1-bd1f-5c25ffc57d70' }
-
   let(:application_data) { JSON.parse(LaaCrimeSchemas.fixture(1.0, name: 'post_submission_evidence').read) }
+  let(:complete_cta) { 'Mark as completed' }
```

Comment:
Taken from the now removed "Mark as complete" spec as PSE is the only application that has this behaviour.